### PR TITLE
Konmari'ing your query migration blog post

### PR DESCRIPTION
**Pull request by krevitt**
## Description & motivation
Blog post from issue https://github.com/dbt-labs/docs.getdbt.com/issues/1760

### CODE REVIEW

File: website/blog/2022-09-08-konmari-your-query-migration.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+Here are a few things to look for:
+
+- Get rid of unused sources  — [there’s a package for that!](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/)
+- Remove columns that are being brought in with import <Term id="cte">CTEs</Term>, but are just clogging your query
+- Only bring in the columns you need (this is especially true for BigQuery and Redshift for performance purposes)
+- Where you can, do the same with rows! Is a filter being applied in the final query, that could be moved to a CTE, or maybe even an intermediate model?
+- Remember that in most cases, it is more performant to filter and truncate the data before the joins take place.
+
+## Steps 4 & 5: Tidy by category & Follow the right order—upstream to downstream
```
Comment (KiraFuruichi) [inline website/blog/2022-09-08-konmari-your-query-migration.md]:
```suggestion
## Steps 4 & 5: Tidy by category and follow the right order—upstream to downstream
```

**Comment by krevitt** [comment]
OK this is ready for a 👍 @KiraFuruichi + @lbenezra-FA ([deploy preview here](https://deploy-preview-1958--docs-getdbt-com.netlify.app/blog/marie-kondo-query-migration)).  

@Lauren, I clipped out the before/after DAG graphics from the intro section (couldn't figure out a way to get those to appear side-by-side with the boxes), lmk if that works or if you'd prefer to make a lockup of those two together in one image.

File: website/blog/2022-09-08-konmari-your-query-migration.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
+---
+title: "KonMari your Data: Planning a Query Migration Using the Marie Kondo Method"
```
Comment (KiraFuruichi) [inline website/blog/2022-09-08-konmari-your-query-migration.md]:
```suggestion
title: "KonMari your data: Planning a query migration using the Marie Kondo method"
```
Just a suggestion for the casing

**Review by KiraFuruichi — APPROVED** [review]
#imageking

**Review by lbenezra-FA — APPROVED** [review]
changed a few little things. looks good!

**Pull request merged**